buildtools/third_party/libc++/BUILD.gn

```diff
@@ -54,6 +54,13 @@ target(_libcxx_target_type, "libc++") {
     "//build/config:common_deps",
     "//third_party/catapult/devil:devil",
   ]
+  if (is_cobalt && is_starboard && current_toolchain == cobalt_toolchain) {
```

Comment:
What starboard APIs does libc++ use? Maybe we don't need this dependency?

Reply:
will revisit